build.gradle

```diff
@@ -985,6 +985,15 @@ project(':iceberg-open-api') {
       exclude group: 'org.apache.commons', module: 'commons-configuration2'
       exclude group: 'org.apache.hadoop.thirdparty', module: 'hadoop-shaded-protobuf_3_7'
       exclude group: 'org.eclipse.jetty'
+      exclude group: 'com.google.re2j', module: 're2j'
```

Comment:
Excluded some more which are unrelated to 'Configuration' class. 

These were not included in the license too.